src/include/miopen/solver.hpp

```diff
@@ -1015,6 +1015,66 @@ struct PerformanceImplicitGemmForwardV4R4Xdlops
     std::tuple<std::size_t, bool> CalculateLdsNumberOfByte(const ConvolutionContext& ctx) const;
 };
 
+struct PerformanceImplicitGemmForwardV4R4Xdlops_Universal
+    : Serializable<PerformanceImplicitGemmForwardV4R4Xdlops_Universal>
+{
+    int GemmMPerBlock;
+    int GemmNPerBlock;
+    int GemmKPerBlock;
+    int GemmMPerWave;
+    int GemmNPerWave;
+    int GemmKPack;
+    int Mfactor;
+    int Nfactor;
+    int Kfactor;
```

Comment:
Please rename them to ```GemmMFactor, GemmNFactor, GemmKFactor``` to differentiate them from Convolution's N and K

Reply:
I don't see they are being tuned. If not, they shouldn't be part of tuning parameters

Reply:
It can tune, but we need to decide how many configs
I am afraid that  someone complain too much configs.....

Reply:
becuase I can't detect we can skip tune or not if we open these parameters ,so it will increase a lot of time

Reply:
Ok then. You can keep them as tuning parameters, just make sure you don't actually tune them for now.

Maybe we will really tune them in the future.